### PR TITLE
Add underscore to enum and bitflags member starting with digits

### LINKED ISSUE #600: Enum generation error for GstMpegts
In the file GstMpegts-1.0.gir there is an enum with specification starting:
```xml
    <enumeration name="DVBCodeRate" c:type="GstMpegtsDVBCodeRate">
      <member name="none" value="0" c:identifier="GST_MPEGTS_FEC_NONE">
      </member>
      <member name="1_2" value="1" c:identifier="GST_MPEGTS_FEC_1_2">
      </member>
      <member name="2_3" value="2" c:identifier="GST_MPEGTS_FEC_2_3">
      </member>
      <member name="3_4" value="3" c:identifier="GST_MPEGTS_FEC_3_4">
      </member>
      <member name="4_5" value="4" c:identifier="GST_MPEGTS_FEC_4_5">
      </member>
```
this generates the output starting:
```rust
#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
pub enum DVBCodeRate {
    None,
    12,
    23,
    34,
    45,
```
which isn't valid Rust code. :-(

Am I just missing an option in using `gir`:
```
gir -d ../gir-files -o . -c ../Gir_GstMpegts.toml
```

### PR DESCRIPTION
**Pull request by EPashkin**
Fix #600

@ceyusa can you test it?

cc @GuillaumeGomez, @sdroege, @russel

### CODE REVIEW

File: src/nameutil.rs [hunk 1]
```diff
@@ -45,6 +44,22 @@ pub fn module_name(name: &str) -> String {
     mangle_keywords(name.to_snake()).into_owned()
 }
 
+pub fn enum_member_name(name: &str) -> String {
+    if name.starts_with(|c: char| c.is_digit(10)) {
```
Comment (sdroege) [inline src/nameutil.rs]: Maybe make this the other way around? If it does not start with a letter

File: src/nameutil.rs [hunk 2]
```diff
@@ -45,6 +44,22 @@ pub fn module_name(name: &str) -> String {
...
+        format!("_{}", name.to_camel())
+    } else {
+        name.to_camel()
+    }
```
Comment (sdroege) [inline src/nameutil.rs]: And maybe error if name is not only containing alpha-num-underscore

File: src/nameutil.rs (hunk 1 above)
Comment (EPashkin) [reply src/nameutil.rs]: You right

File: src/nameutil.rs (hunk 2 above)
Comment (EPashkin) [reply src/nameutil.rs]: IMHO this unneeded, these members better ignored manually

Comment (sdroege) [reply src/nameutil.rs]: ok :)

**Comment by ceyusa** [comment]
it works! or at least, it generates a compileable code :) thanks!

**Comment by sdroege** [comment]
Let's go for that then :)

**Comment by GuillaumeGomez** [comment]
Let's go for it then!

**Pull request merged**